camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/catalog/SpecParameterUnwrappingTest.java

```diff
@@ -146,8 +151,9 @@ public void testRootParametersUnwrapped() {
         CatalogItem<?, ?> item = catalog.getCatalogItem(SYMBOLIC_NAME, TEST_VERSION);
         AbstractBrooklynObjectSpec<?,?> spec = catalog.peekSpec(item);
         List<SpecParameter<?>> params = spec.getParameters();
-        assertEquals(params.size(), NUM_APP_DEFAULT_CONFIG_KEYS + 1, "params="+params);
         assertTrue(Iterables.tryFind(params, nameEqualTo("simple")).isPresent());
+        assertTrue(Iterables.tryFind(params, nameEqualTo(SHARED_CONFIG.getName())).isPresent());
+        assertEquals(params.size(), NUM_APP_DEFAULT_CONFIG_KEYS + 2, "params="+params);
```

Comment:
nice idea to use constants for all the counts.  have also added comment about unwrapping, agree it's a bit wasteful, but doesn't seem to do any harm.